docs/process/bazel.md

```diff
@@ -42,6 +42,24 @@ Since the requirement is quite lax on Linux but quite strict on windows, what en
 happening is that lack of `require.resolve` calls go unnoticed until someone tries to run
 things on Windows, at which point it breaks.
 
+## Debugging jasmine_node_test
+
+On Linux, Bazel tests will run under a sandbox for isolation.
+You can turn off this sandbox by adding the [`local = True`](https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes-tests) attribute to the rule.
```

Comment:
NB: for a few tests, sandbox is required as otherwise the rules_nodejs linker symlinks will conflict with the yarn workspace symlinks in node_modules

Reply:
Added note.